src/core/matching_state.rs

```diff
@@ -1,78 +1,114 @@
-use super::{Entry, Heuristic, LangData, MatchData, Sender};
+use super::*;
```

Comment:
I don't like using `*`...

Reply:
Too bad. This file uses nearly all of the imports and structs defined in its parent module, so that way it looked cleaner to me.

Reply:
I don't agree but ok.